test/acceptance/registry.js

```diff
@@ -46,6 +51,136 @@ describe('registry', function(){
     });
   });
 
+  describe('GET /hello-world-custom-headers', function() {
+
+    var httpGetRequest = function(options, callback) {
+      var callbackDone = false;
+      var requestData = url.parse(options.url);
+      requestData.headers = options.headers || {};
+      requestData.method = 'get';
+
+      var req = require('http').request(requestData).on('response', function(response) {
```

Comment:
I guess you're doing all of this because `minimal-request` doesn't allow you to access response headers. This is not great, I can totally extend minimal-request to do that and then this test should look a bit more neat. Still, not a blocker for merging this, definitely a quick improvement for a later PR. Sorry for that 👍